utils/packets/__main__.py

```diff
@@ -5,7 +5,12 @@
 
 from utils.db.helpers import snowflake_engine
 from utils.helpers import initialize_cli_logging
-from utils.packets.service import ingest_support_packet, ingest_survey_packet
+from utils.packets.service import (
+    ingest_support_packet,
+    ingest_support_packets_from_s3,
```

Comment:
I assume there is no need to add tests for `ingest_support_packets_from_s3` (only the table name changes).

Reply:
It's calling the same function as `ingest_surveys_from_s3`. The individual components have been tested. We can add an end-to-end test if you prefer. However, since there might be changes depending on the structure of the S3 bucket, I prefer to wait for them before adding the extra data.